src/mds/MDBalancer.cc

```diff
@@ -215,31 +212,30 @@ double mds_load_t::mds_load()
   return 0;
 }
 
-mds_load_t MDBalancer::get_load(utime_t now)
+mds_load_t MDBalancer::get_load()
 {
-  mds_load_t load(now);
+  auto now = clock::now();
+
+  mds_load_t load(mds->mdcache->decayrate);
```

Comment:
I think it's better to use zero decay rate here, because mds loads from other mds  have zero decay rate

Reply:
Good catch!